Content.Shared/GameObjects/Components/Observer/SharedGhostComponent.cs

```diff
@@ -45,7 +46,7 @@ public class GhostWarpRequestMessage : ComponentMessage
     {
         public EntityUid PlayerTarget;
         public string WarpName;
-        public GhostWarpRequestMessage(EntityUid playerTarget = default, string warpTarget = default)
+        public GhostWarpRequestMessage(EntityUid playerTarget = default, string warpTarget = default!)
```

Comment:
Should be handled

Reply:
Smug will fix this seperately, and I will rollback this file and merge this PR with master.